shhext/service_test.go

```diff
@@ -0,0 +1,88 @@
+package shhext
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/node"
+	"github.com/ethereum/go-ethereum/p2p"
+	whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestShhExtSuite(t *testing.T) {
+	suite.Run(t, new(ShhExtSuite))
+}
+
+type ShhExtSuite struct {
+	suite.Suite
+
+	nodes    []*node.Node
+	services []*Service
+	whisper  []*whisper.Whisper
+}
+
+func (s *ShhExtSuite) SetupTest() {
+	s.nodes = make([]*node.Node, 2)
+	s.services = make([]*Service, 2)
+	s.whisper = make([]*whisper.Whisper, 2)
+	port := 32139
+	for i := range s.nodes {
+		i := i
```

Comment:
I know this trick to use `i` inside the closure, but I would prefer something like `nodeIdx := i`. This `foo := foo` always looks strange. 😅

Reply:
`i := i` is so short though, i will add a comment